README.md

```diff
@@ -86,18 +86,25 @@ Name of the team that is the main point of contact for information about this AP
 
 ### type
 
-API type - SOAP, REST, HL7, GraphQL, etc.
+API type: `SOAP`, `SOAP/HL7`, `HL7`, `REST`, `REST/FHIR`, `Solr`, or `GraphQL`
 
 ### status
 
-The status of an API is determined by availability.
+Status indicating the level of stability and support, defined as follows:
 
-- **Live**: in Production.
-- **Beta**: in Production but for pilot use only.
-- **Alpha**: in SIT but not in production.
-- **Experimental**: in internal Dev environment or Sandbox but not in SIT or production.
+- `experimental`: Proof of Concept or Prototype still under development.
+- `alpha`: In a test (SIT) environment and undergoing formal evaluation by a tester
```

Comment:
I don't see any added value in adding the abbreviation _SIT_, particularly as it's not explained or used elsewhere.

Reply:
Yeah, I was unsure about whether it's needed, deleted and added a few times, so happy to delete it again.  Although as an abbreviation I think it's fine as it's a commonly used term in software development, not specific to NWIS (maybe I should have called it WSIT, now there a thought...)

```suggestion
- `alpha`: In a test environment and undergoing formal evaluation by a tester
```